tutoraspects/plugin.py

```diff
@@ -44,7 +44,7 @@
                 "edx-event-routing-backends==v7.0.1",
             ],
         ),
-        ("EVENT_SINK_CLICKHOUSE_MODELS", ["course_overviews", "user_profile"]),
+        ("EVENT_SINK_CLICKHOUSE_MODELS", ["course_overviews", "user_profile", "external_id"]),
```

Comment:
This should be a PII model, otherwise lgtm

Reply:
Yes, it needs to be in both lists or else it will not initialize it.

I think we just need to update the logic in the init job to separate both concerns